source/tutorial/triggers-fts.txt

```diff
@@ -0,0 +1,345 @@
+might appear in your important or time-sensitive tasks.
+
+.. procedure::
+
+   .. step:: Create the Alert Box Component
+
+      In the template app's ``src/`` directory, create a new ``AlertBox.tsx`` file 
+      and add the following code. This file contains the UI form that lets you 
+      enter the terms to be alerted on.
+
+      .. literalinclude:: /tutorial/backend-examples/AlertBox.tsx
+         :language: javascript
+         :caption: src/AlertBox.tsx
+
+   .. step:: Update the Main View
+
+      In ``src/ItemListView.tsx``, add the following import to the top of the file:
+
+      .. code-block:: javascript
+      
+         import {AlertBox} from './AlertBox';
+
+      Then, add the following lines to your code to render a button that 
+      displays the alert box when clicked:
+
+      .. literalinclude:: /tutorial/backend-examples/ItemListView.snippet.showAlertBox.tsx
+         :language: javascript
+         :caption: src/ItemListView.tsx
+         :emphasize-lines: 2
+
+      .. literalinclude:: /tutorial/backend-examples/ItemListView.snippet.alertOverlayButton.tsx
+         :language: javascript
+         :caption: src/ItemListView.tsx
+         :emphasize-lines: 19-41
+
+      To change the size and color of the button, add the following lines to your code:
+
+      .. literalinclude:: /tutorial/backend-examples/ItemListView.snippet.alertButtonStyle.tsx
+         :language: javascript
+         :caption: src/ItemListView.tsx
+         :emphasize-lines: 7-11
+
+Store Alert Terms in Atlas
+--------------------------
+
+Now that you've created the frontend component for the alert box, 
+configure the application's backend to store and keep track of your 
+alert terms in Atlas.
+
+.. procedure::
+
+   .. step:: Create a New Collection
+
+      Create a collection to store the terms that users enter in the app.
+      In the Atlas UI: 
+
+      A. Click the :guilabel:`Data Services` tab.
+      
+      #. For the deployment that's synced to the template app, 
+         click :guilabel:`Browse Collections`.
+      
+      #. In the left navigation, click the :guilabel:`+` icon next to the ``todo`` 
+         database to add a new collection.
+      
+      #. Name the collection ``alerts``, then click :guilabel:`Create` to save 
+         the collection.
+
+   .. step:: Enable Access to the Collection
+
+      After creating the collection, you must give your app the necessary permissions
+      to write to the ``todo.alerts`` collection:
+
+      A. Click the :guilabel:`App Services` tab.
+      
+      #. Click the tile for your app.
+
+      #. In the left navigation under :guilabel:`Data Access`, click :guilabel:`Rules`.
+
+      #. Under the ``todo`` database, click the ``alerts`` collection.
+      
+      #. In the right dialog box, select the :guilabel:`readAndWriteAll` preset role.
+      
+      #. Click :guilabel:`Add preset role` to confirm your selection.
```

Comment:
Might want to add a blurb that if you're using deployment drafts in the UI then you need to deploy before the changes are live. (I'm not actually sure if these are turned on by default for the RN template backend)

Reply:
Ah, I had development mode on so drafts were disabled - good catch!